packages/website/docs/using-query-suggestions-plugin.md

```diff
@@ -2,7 +2,137 @@
+
+```diff
+import { autocomplete } from '@algolia/autocomplete-js';
++ import { createQuerySuggestionsPlugin } from '@algolia/autocomplete-plugin-query-suggestions';
++
++ const searchClient = algoliasearch(
++   'yourAppID',
++   'yourSearchApiKey'
++ );
++
++ const querySuggestionsPlugin = createQuerySuggestionsPlugin({
++   searchClient,
++   indexName: 'yourQuerySuggestionsIndexName',
++   },
++ });
++
+autocomplete({
+  container: '#autocomplete',
+-  plugins: [],
++  plugins: [querySuggestionsPlugin],
+  openOnFocus: true,
+});
+```
+
+You can optionally pass a `getSearchParams` function to apply [Algolia query parameters](https://www.algolia.com/doc/api-reference/api-parameters/) to the suggestions returned from the plugin. This is particularly useful if you need to align your Query Suggestions with other sections displayed in the autocomplete, like [recent searches](using-recent-searches-plugin).
+
+```diff
+import { autocomplete } from '@algolia/autocomplete-js';
+import { createQuerySuggestionsPlugin } from '@algolia/autocomplete-plugin-query-suggestions';
+
+const searchClient = algoliasearch(
+  'yourAppID',
+  'yourSearchApiKey'
+);
+
+const querySuggestionsPlugin = createQuerySuggestionsPlugin({
+  searchClient,
+  indexName: 'yourQuerySuggestionsIndexName',
++ getSearchParams() {
++   return recentSearchesPlugin.data.getAlgoliaSearchParams({
++     hitsPerPage: 5,
++   });
+  },
+});
+
+autocomplete({
+  container: '#autocomplete',
+  plugins: [querySuggestionsPlugin],
+  openOnFocus: true,
+});
+```
+This creates a basic Query Suggestions implementation. Try it out below:
+
+<AutocompleteExample
+  plugins={[querySuggestionsPlugin]}
+  openOnFocus={true}
+/>
+
+:::note
+
+These suggestions are based on a [public dataset of BestBuy products](https://github.com/algolia/datasets/tree/master/ecommerce).
+
+:::
+
+## Next steps
+
+This tutorial focuses on adding Query Suggestions to an autocomplete menu. Many autocomplete menus also include recent searches and possibly other items. You can find an example including [recent searches](using-recent-searches-plugin) and [static predefined items](sources#using-static-sources) on [GitHub](https://github.com/algolia/doc-code-samples/tree/autocomplete-v1/Autocomplete/multi-source). To learn how to display multiple sections in one autocomplete, read the [guide on creating multi-source autocompletes](creating-multi-source-autocompletes).
```

Comment:
Curious if the plugin support QS categories yet? If not, might be worth adding this as a possible next step, but one they can't rely on the plugin for yet.

Reply:
It doesn't yet. It requires quite some code user-land right now so perhaps we can stay silent about it until they ask for it or that we support it?

We created an example in the official docs but that's a use case that we should cover soon in the plugin anyway IMO.